### ISSUE TITLE
Incorrect handling of japanese path separators

### ISSUE THREAD

**Issue by mrhso** [dscharrer/innoextract#101]
https://tasofro.net/arc/th105_update_106a.exe
`innoextract --codepage 932 th105_update_106a.exe` doesn't work for filenames.
```
Extracting "東方緋想天 Ver1.06a" - setup data version 5.2.3
 - "app\th105.exe"
 - "app\th105c.dat"
 - "app\readme.txt"
 - "app\.txt"
 - "app\configex.ini"
Done.
```

**Comment by dscharrer** [comment]
Doesn't seem to be an encoding issue but something that goes wrong while expanding the paths - you can see the correct raw paths with `--dump`:
````
Testing "東方緋想天 Ver1.06a" - setup data version 5.2.3
 - "{app}¥th105.exe"
 - "{app}¥th105c.dat"
 - "{app}¥readme.txt"
 - "{app}¥マニュアル正誤表.txt"
 - "{app}¥configex.ini"
Done.
````

**Comment by dscharrer** [comment]
This should be fixed with [development snapshot 1.9-dev-2020-01-12](https://constexpr.org/innoextract/files/snapshots/innoextract-1.9-dev-2020-01-12/).

**Comment by mrhso** [comment]
There is 'CP932' and 'Shift-JIS' in libiconv. Which should it use?
